lib/l10n/tr_nums.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:io';
+import 'package:intl/intl.dart';
+
+String trNums(num number) {
+  final formatter = NumberFormat('###.#', Platform.localeName);
```

Comment:
Get locale from `context` not from `Platform`

Reply:
context returns 'ar' while we need 'ar_EG', [country code is missing].